docs/user-guide/browser-profiles.md

```diff
@@ -14,17 +14,38 @@ Browser Profiles are saved instances of a web browsing session that can be reuse
 
     - Due to nature of social media specifically, existing accounts may have personally identifiable information, even when accessing otherwise public content.
 
-    Of course, there are exceptions — such as when the goal is to archive personalized or private content accessible only from designated accounts.
+    Of course, there are exceptions — such as when the goal is to archive personalized or private content accessible only from designated accounts. In these instances we recommend changing the account's password after crawling is complete.
 
 ## Creating New Browser Profiles
 
-New browser profiles can be created on the Browser Profiles page by pressing the _New Browser Profile_ button and providing a starting URL. Once in the profile creator, log in to any websites that should behave as logged in while crawling and accept any pop-ups that require interaction from the user to proceed with using the website.
+New browser profiles can be created on the Browser Profiles page by pressing the _New Browser Profile_ button and providing a starting URL. 
 
 Press the _Finish Browsing_ button to save the browser profile with a _Name_ and _Description_ of what is logged in or otherwise notable about this browser session.
 
+### Logging into Websites
+
+To crawl content as a logged in user, log into the website you wish to archive as you would on any other browser. Once the account has been logged in, that's it!
+
+### Accepting Popups
+
+Some websites are required to get informed consent from users to track them, others require their users to verify their age before viewing adult content. While both of these could be handled with logins, often websites choose to use cookies — small pieces of configuration data stored in the browser. Interacting with popups that store the user's choices in a cookie will in turn store those cookies within the browser profile. Like everything else those cookie values will be used when crawling with the browser profile.
+
+### Changing Browser Settings
+
+Browser profiles don't just affect websites! Any of Brave's settings (available at the URL `brave://settings/`) set in the profile creator will be used while crawling.
+
+??? example "Example: Blocking page resources with Brave's Shields"
+    Whereas the crawler's scoping settings can be used to define which pages should be crawled, Brave's [Shields](https://brave.com/shields/) feature can block resources on pages from being loaded. By default, Shields will block [EasyList's cookie list](https://easylist.to/) but it can be set to block a number of other included lists under Brave `Settings > Shields > Filter Lists`.
```

Comment:
I don't think it does use the Easylist cookie list by default, pretty sure we need to enable that in Shields manually if people want to use it to avoid GDPR popups.

Reply:
On the other hand, some of Brave's shields such as moderate ad blocking are enabled by default, and maybe we want to point that out here.

Reply:
These were the defaults I checked?  If that's not how it works in practice maybe not a good thing to note, but I did note if for that reason.